src/content/developers/docs/spin-up-your-node/index.md

```diff
@@ -0,0 +1,146 @@
+Some clients offer builds for different architectures which makes possible to run it on [SCBs with ARM](/en/developers/docs/nodes-and-clients/#ethereum-on-a-single-board-computer/). Another easy way is using [Ethbian](https://ethbian.org/index.html) operating system for Raspberry Pi 4 which enables [run Ethereum client by simply flashing SD card](/en/developers/tutorials/run-node-raspberry-pi/).
+Depending on the chosen software and the hardware, the initial synchronization time and storage requirements may vary. Check [orientational table](/en/developers/docs/nodes-and-clients/#recommended-specifications) in Nodes and clients page. 
+Also make sure your internet connection is not limited by [bandwidth cap](https://en.wikipedia.org/wiki/Data_cap). It is recommended to use unmetered connection since initial sync and network gossip could exceed limits of your ISP. 
+
+#### Operating system {#operating-system}
+
+All clients support major operating systems - Linux, MacOS, Windows. This enables to run client on regular desktop or server machine with OS that suits user the best. Make sure OS is up to date to avoid potential issues and security vulnerabilities. 
+
+
+## Spinning up the node {#spinning-up-node}
+
+### Getting the client software {#getting-the-client}
+
+First, let's download chosen client software itself. 
+
+Users can simply download executable application or installation package which suits their operating system and architecture. It is always recommended to verify signatures and checksums of downloaded packages. Some clients also offer repositories for easier installation and updates. 
+Developers might prefer building from source. All of the clients are open source and it is easy to build them from source code with proper compiler. 
+
+Stable mainnet client implementations, summarized on page above, can be downloaded from their release pages - [Geth](https://geth.ethereum.org/downloads/), [OpenEthereum,](https://github.com/openethereum/openethereum/releases) [Nethermind](https://downloads.nethermind.io/), [Besu](https://pegasys.tech/solutions/hyperledger-besu/). 
+
+### Starting the client {#starting-the-client}
+
+Before starting Ethereum client software, perform last check that the environment is ready for deploying it. For example, make sure:
+- There is enough disk space considering chosen network and sync mode.
+- Memory and CPU is not halt by other programs.
+- Operating system is updated to latest version.
+- System has correct time and date.
+- Make sure your router and firewall accepts connection on listening ports. By default Ethereum clients use a listener (TCP) port and a discovery (UDP) port, both on 30303 by default.
```

Comment:
Don't need "make sure"

"Your router and firewall accept connections on listening ports. By default Ethereum clients use a listener (TCP) port and a discovery (UDP) port, both on 30303 by default."